src/main/java/org/b612foundation/adam/opm/.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can you add .vs/ to .gitignore?

Reply:
I'm not using Visual Studio for this project, but I think it started automatically and created the .vs folder... I'm thinking I can just delete it... instead of adding to gitignore -thoughts?